1-mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,4 +1,24 @@
-function setAlarm() {}
+const setAlarm = () => {
+  const inputEl = document.getElementById("alarmSet");
+  let userInputValue = inputEl.value;
+  const h1TimeRemainingEl = document.getElementById("timeRemaining");
+
+  let timer = setInterval(() => {
+    let minutes = Math.floor(userInputValue / 60);
+    let seconds = Math.floor(userInputValue % 60);
+
+    minutes = minutes < 10 ? "0" + minutes : minutes;
+    seconds = seconds < 10 ? "0" + seconds : seconds;
+
+    h1TimeRemainingEl.innerText = `Time Remaining: ${minutes} : ${seconds}`;
```

Comment:
Because we set this text for the first time after 1000 microseconds, there is a 1 second delay between setting the alarm and when you see it on screen for the first time.